gnucash/gnome/gnucash.appdata.xml.in.in

```diff
@@ -19,7 +19,8 @@
       <li>Keep your small business' accounting up to date</li>
       <li>Create accurate reports and graphs from your financial data</li>
       <li>Set up scheduled transactions to avoid repeated data entry</li>
-      <li>Exchange by CSV/FinTS(former HBCI) or import SWIFT-MT9xx/QIF/OFX data including Transaction Matching</li>
+      <li>Process your bank data via online banking using OFX DirectConnect, FinTS or YelloNet</li>
+      <li>Import data in various formats such as CSV/SWIFT-MT9xx/QIF/OFX data including transaction matching</li>
```

Comment:
It is then missing
<li>Export several data (accounts, transactions, […]) as CSV</li>

Reply:
Perhaps each list enry should end with `;` with exception of the last `.`